### PR TITLE
zuul: use bigger image

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
Depends-On: https://github.com/wazo-platform/sf-config/pull/36

File: zuul.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@
     check:
       jobs:
         - wazo-acceptance-job:
-            nodeset: debian10-vm
+            nodeset: debian10-vm-m2
     gate:
       jobs:
         - wazo-acceptance-job:
-            nodeset: debian10-vm
+            nodeset: debian10-vm-m2
```
Comment (fblackburn1) [inline zuul.yaml]: Do we really need `-m2` with all optimization done on linphone docker?

**Comment by fblackburn1** [comment]
I've cherry-picked the first commit of whitelist docker to master

**Comment by sileht** [comment]
recheck

File: zuul.yaml (hunk 1 above)
Comment (sileht) [reply zuul.yaml]: Currently the VM have 2G and only between 250Mo/300Mo free RAM when everything is installed.

Comment (sileht) [reply zuul.yaml]: And we still see that job fail with out of memory issue

**Comment by sileht** [comment]
recheck

**Comment by sileht** [comment]
Depends-On: https://github.com/wazo-platform/sf-jobs/pull/26

**Comment by sileht** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
Depends-on: https://github.com/wazo-platform/sf-jobs/pull/30

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Pull request merged**